source/installation-guide/installing-wazuh-agent/wazuh_agent_rpm.rst

```diff
@@ -83,19 +83,34 @@ SUSE 11
 Installing Wazuh agent
 ----------------------
 
-1. On your terminal, install the Wazuh agent as follows:
+1. On your terminal, install the Wazuh agent. You can choose only installation or an installation with a registration / configuration included:
 
-  * Using the ``yum`` package manager:
+  a) Only installation:
 
-    .. code-block:: console
+     * Using the ``yum`` package manager:
+   
+       .. code-block:: console
+   
+         # yum install wazuh-agent
+   
+     * Using the ``zypper`` package manager:
+   
+       .. code-block:: console
+   
+         # zypper install wazuh-agent
+         
+     .. note:: Now that the agent is installed, the next step is to register and configure it to communicate with the manager. For more information about this process, please visit the :doc:`user manual<../../user-manual/agents/registering/index>`.
 
-      # yum install wazuh-agent
+  b) Installation with a registration / configuration included:
 
-  * Using the ``zypper`` package manager:
+    You can automate the agent registration and configuration using variables. 
 
     .. code-block:: console
 
-      # zypper install wazuh-agent
+      # WAZUH_MANAGER_IP="192.168.1.2" yum install wazuh-agent 
```

Comment:
Maybe, it would be interesting to add an example of the automatic registration and configuration using `zypper` too. What do you think about this @alberpilot ?

Reply:
I think that the previous section explains the difference between both package manager. Although we will take into account this suggestion in the next revision. Thanks.

Reply:
Got it :smile: